ansible/files/rock_start

```diff
@@ -40,4 +40,9 @@ systemctl status broctl | egrep "^\s*Active"
 #sleep 5
 #systemctl status stenographer | egrep "^\s*Active"
 
+echo "Starting FSF..."
+systemctl start fsf
```

Comment:
I'd like to make these conditional, since this will break of the user didn't install fsf.

Something like (untested):

```bash
if grep '^with_stenographer: True' /etc/rocknsm/config.yml; then
    systemctl start fsf
    sleep 5
    systemctl status fsf | egrep "^\s*Active"
fi
```